crates/web5/src/lib.rs

```diff
@@ -2,9 +2,10 @@ pub mod credentials;
 pub mod crypto;
 pub mod dids;
 
+mod datetime;
 pub mod errors;
+pub mod jose;
```

Comment:
i understand that we're trying to remove josekit as a dependency, but we didn't previous re-export josekit's API. why are we exposing a jose API now? i thought this PR is meant to change the web5 crate's implementation without changing the API

Reply:
yeah, let's not quite make this public... yet. let's consider later after further deliberation. I think we probably will want to make it public, though that doesn't mean include it in the APID (though it may), primarily on two use cases: tbdex (which has a detached compact jws as it's message signatures) and the web5 cli (which, it's common place among devrel to want to inspect JWTs, especially when working with KCC matters). if we were to make this public, the messaging would be clear, that this is not intended to be a JOSE library we're building, but it _does_ offer JOSE features unique to web5 use cases (for example, web5 makes a requirement of the JOSE header kid being a DID VM id).

nevertheless, for now, I will not make it public